BrainPortal/app/models/cluster_task.rb

```diff
@@ -1481,9 +1480,10 @@ def unarchive_work_directory
     Dir.chdir(full) do
 
       if ! File.exists?(tar_file)
+        self.addlog("Cannot unarchive: tar archive #{tar_file} does not exist.")
```

Comment:
This message will be printed even if the tar archive IS found two lines below. I think that's misleading. The original code with a single 'addlog()' statement was better. So please undo this (remove new line 1483 and revert new line 1486)

Reply:
I am still waiting for this to be addressed

Reply:
rolled back